### ISSUE TITLE
Training Time and training dataset

### ISSUE THREAD

**Issue by MaureenZOU** [amjltc295/Free-Form-Video-Inpainting#29]
Hi Authors : )

Thanks for your interesting work. It would be really appreciated if you could give some information about your work on training epochs and how do you select the video for training!

Best,
Xueyan

**Comment by amjltc295** [comment]
We use 1.6k videos from YouTube-VOS in the FVI training set in training. Train until the model is converged (could see that in the tensorboard).

**Comment by MaureenZOU** [comment]
Got it with thanks! But could you please give a rough estimation of how many epochs you have trained? As the default number is 1000, which seems too long. Thanks!

**Comment by amjltc295** [comment]
I forgot the exact number, but it should be <40, depending on the learning rate.

**Comment by amjltc295** [comment]
I forgot the exact number, but it should be <200, depending on the learning rate.

**Comment by MaureenZOU** [comment]
Got it with thanks!

**Comment by MaureenZOU** [comment]
Got it thanks! BTW, after reading part of your code, I think they are nice structured and clean.
